### ISSUE TITLE
"Small changes", review culture best practice

### ISSUE THREAD

**Issue by Beanow** [sourcecred/docs#7]
Tracking suggestion from here: https://github.com/sourcecred/docs/pull/1#discussion_r368150278

> @vsoch 
> And for this reason, it's suggested to favor more, smaller pull requests over a single, larger one. (or something like that).

> @decentralion 
> Yep. We may want a "best practices for contributors" section that we can mention:
> * ask the community for feedback early in the process (e.g. "is this change a good idea"?)
> * make small changes
> * do self-review
> 
> Want to submit a followon PR? (In the spirit of #2)

**Comment by vsoch** [comment]
Yes I'd be happy to add this on! I'll wait for #9 to go through, not because git can't handle it, but because my brain likes to juggle only a few things at a time :)
